styles.css

```diff
@@ -36,13 +38,14 @@ main {
 }
 
 .nav-bar {
```

Comment:
- [ ] For a better user experience, Kindly make sure your page is not overlapping on the mobile version. The page has to fit on the screen, the user does not need to scroll to see the whole content on the page.
`Hint: You may consider setting the width to 100%`

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201392450-5d1efa1a-e118-41aa-a91c-1dfed8b5584e.png)
![image](https://user-images.githubusercontent.com/97100091/201392497-1f77a648-712a-4c59-bb30-b2e5fc6ac66c.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201392619-12e0779a-e5c1-45ee-81c3-b179795e65dc.png)